.rubocop.yml

```diff
@@ -791,6 +791,7 @@ Style/PredicateName:
   # helpers in the form of `have_something` or `be_something`.
   Exclude:
     - '**/spec/**/*'
+    - '**/test/**/*'
```

Comment:
But these files are exposed by the gems, for plugin testing. So renaming is backwards incompatible in the sense that it will break requires like [this one](https://github.com/AjuntamentdeBarcelona/decidim-barcelona/blob/090d7f5dcd056e72fdb8b3cb68daddab5b7c2fa6/decidim-debates/spec/factories.rb#L2). Are we ok with that?